src/dashboard/server/api/validator/index.js

```diff
@@ -7,6 +7,7 @@ validator.addSchema(require('./endpoints.schema'), 'endpoints')
 validator.addSchema(require('./job.schema'), 'job')
 validator.addSchema(require('./template.schema'), 'template')
 validator.addSchema(require('./priority.schema'), 'priority')
+validator.addSchema(require('./exemption.schema'), 'isExempted')
```

Comment:
We'd better use same names in resource name and field name instead of `exemption` and `isExempted`, at least, the second parameter stands for the schema name of the exemption resource, should be `exemption`